core/src/components/badge/badge.ionic.scss

```diff
@@ -174,3 +175,32 @@
   min-width: globals.$ion-scale-400;
   height: globals.$ion-scale-400;
 }
+
+// Badge Inside Tab Button
+// --------------------------------------------------
+
+:host([vertical].in-tab-button) {
+  position: relative;
+
+  min-width: badge-ionic-vars.$badge-ionic-in-tab-button-min-width;
```

Comment:
This isn't needed. I was able to remove it on my end and verified that the sizes are still correct since we are using the correct `size="xxsmall"` prop.

```suggestion
```

Reply:
This is not needed indeed due the styling of the empty already contains the min-width.
Forgot to remove this. Thank you!
Done!